packages/marko-web-identity-x/browser/custom-column.vue

```diff
@@ -200,11 +215,25 @@ export default {
       type: Boolean,
       default: false,
     },
+    className: {
+      type: String,
+      default: '',
+    },
   },
   /**
    *
    */
   computed: {
+    countryCode() {
```

Comment:
You should do the same for postal code, if you are making the field show/hide based on the selected country.